services/shotgrid_common/utils.py

```diff
@@ -1359,6 +1411,41 @@ def create_new_ayon_entity(
         sg_ay_dict["attribs"].get(SHOTGRID_TYPE_ATTRIB, "")
     )
 
+    status = sg_ay_dict.get("status")
+    if status:
+        # Entity hub expects the statuses to be provided with the `name` and
+        # not the `short_name` (which is what we get from SG) so we convert
+        # the short name back to the long name before setting it
+        status_mapping = {
+            status.short_name: status.name for status in
+            entity_hub.project_entity.statuses
+        }
+        new_status_name = status_mapping.get(status)
+        if not new_status_name:
+            log.warning(
+                "Status with short name '%s' doesn't exist in project", status
+            )
+        else:
+            try:
+                # INFO: it was causing error so trying to set status directly
+                ay_entity.status = new_status_name
+            except ValueError as e:
+                # `ValueError: Status ip is not available on project.`
+                # NOTE: this doesn't really raise exception?
+                log.warning(f"Status sync not implemented: {e}")
+
+    assignees = sg_ay_dict.get("assignees")
+    if assignees:
```

Comment:
Removement of assignees should be propagated too, or?
```suggestion
    if assignees is None:
```

Reply:
This is creation of new User in AYON, there is no resetting here.

Reply:
Not creation of new "User", but I understand... Make sense.